Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -940,7 +940,9 @@ def test_digits(self):
 
     def test_from(self):
         widget = self.create()
-        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=float_round)
+        # A change in Tk 8.6.10 means `from` is not rounded (bpo-41306)
+        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1,
+                             conv=float_round if get_tk_patchlevel() < (8, 6, 10) else noconv)
```

Comment:
This PR duplicates part of #22405, where I produced the same effect with the equivalent of the following.  To me it is easier to read. 

```suggestion
        conv = float_round if get_tk_patchlevel() < (8, 6, 10) else noconv
        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=conv)
```